source/install/install-kubernetes-mattermost.rst

```diff
@@ -113,6 +115,130 @@ pointing to the address you just copied. For example, on AWS you would do this w
 
 Navigate to the ``ingressName`` URL in your browser and use Mattermost.
 
+
+Configuring external database and filestore (Recommended)
+----------------------------------------------------------
+
+When installing Mattermost with Mattermost Operator for production scenario it is recommended to use an external database and filestore.
+
+**1. Create database Secret**
+
+The database Secret needs to be created in the namespace that will hold the Mattermost Installation.
+The Secret should contain the following data:
+
+.. csv-table::
+    :header: "Key", "Description", "Required"
+
+    "DB_CONNECTION_STRING", "Connection string to the database.", "true"
+    "MM_SQLSETTINGS_DATASOURCEREPLICAS", "Connection string to read replicas of the database.", "false"
+    "DB_CONNECTION_CHECK_URL", "The URL used for checking that the database is accessible.", "false"
```

Comment:
Might be worth adding a note here that `DB_CONNECTION_CHECK_URL` is often the same as `DB_CONNECTION_STRING`

Reply:
There is a note below example secret https://github.com/mattermost/docs/pull/4113/files#diff-ac678edad9006c078b03da9e586c48bd39bb15913a0a56e1d6f1852846148ce4R150
Not sure if you have missed it or you prefer to add this information to the table?

Reply:
Oh yeah I missed that 👍